_episodes/docker-desktop.md

```diff
@@ -0,0 +1,411 @@
+  </article>
+</div>
+</div>
+<br/>
+{::options parse_block_html="false" /}
+
+The container does not need to stay alive forever, and you can see that there is a 'stop' icon on the top right.
+If we stop the container, we get a familiar empty tab in 'Terminal' and 'Stats'.
+The 'Containers' tab on the left will also show the container status as 'Exited'
+
+{::options parse_block_html="true" /}
+<div>
+<ul class="nav nav-tabs nav-justified" role="tablist">
+  <li role="presentation" class="active">
+    <a href="#gs_stopped_Terminal" aria-controls="gs_stopped_Terminal" role="tab" data-toggle="tab">Terminal</a>  </li>
+  <li role="presentation">
+    <a href="#gs_stopped_Stats" aria-controls="gs_stopped_Stats" role="tab" data-toggle="tab">Stats</a>  </li>
+  <li role="presentation">
+    <a href="#gs_stopped_Containers" aria-controls="gs_stopped_Containers" role="tab" data-toggle="tab">Containers</a>  </li>
+</ul>
+<div class="tab-content">
+  <article role="tabpanel" class="tab-pane active" id="gs_stopped_Terminal">
+  ![Terminal tab in container from stopped getting-started image.](../fig/docker-desktop/26_gettingstarted_exited_t.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_stopped_Stats">
+  ![Stats tab in container from stopped getting-started image.](../fig/docker-desktop/27_gettingstarted_exited_s.png)
+  </article>
+  <article role="tabpanel" class="tab-pane" id="gs_stopped_Containers">
+  ![Container list after stopping getting-started image.](../fig/docker-desktop/28_gettingstarted_exited_c.png)
+  </article>
+</div>
+</div>
+<br/>
+{::options parse_block_html="false" /}
+
+Now lets say I want to run the `getting-started` image again. So I go to the 'Images' tab, and click run.
+Now lets go to the 'Terminal' tab, and try and find our directory with `ls`. The directory is not there.
+We'd also installed `htop`. so lets have a go at running it. Not there either.
+![Terminal in fresh getting-started image.](../fig/docker-desktop/29_gettingstarted_2.png)
+
+If this is not surprising to you, it means you're getting the hang of it.
+When we re-ran the *image*, we created a **new** *container*.
+The new container is created from the template saved in the image, and so 'our' changes have banished.
+This becomes very clear when we go back to the 'Containers' tab on the left.
+We can see that the first container we created from the `getting-started` image is there, next to
+the new container (which is still running, by the way).
+![Containers after new run of getting-started image.](../fig/docker-desktop/30_gettingstarted_2_containers.png)
+
+## Reviving containers
+We *can* get the old container running again, although this is rarely something we'd *want* to do.
+In docker desktop, all we need to do is click on the 'Start' button from the 'Containers' list.
+The terminal will appear empty, because it is a new session, but you will even be able to 'recall' commands.
+### TO DO: Change to tabs instead of gifs?
```

Comment:
better to put an issue in than have TO DO for me

Reply:
Those were actually just markers for you, so that you could consider whether a tab or a gift would be better... so I have just removed them...